src/components/SaveAlertPopup.vue

```diff
@@ -0,0 +1,140 @@
+<script lang="ts">
```

Comment:
For global reactive : 
exemple 
```
export const globalCount = reactive({
  count: 0,
  increment() {
    this.count++;
  },
  reset() {
    this.count = 0;
  },
});
```

import the globalCount and use attribute or method like any object